src/stringProcessing/markWordsInSentences.js

```diff
@@ -23,24 +23,11 @@ const collectMarkingsInSentence = function( sentence, topicFoundInSentence ) {
 export function markWordsInSentences( wordsToMark, sentences, locale ) {
 	let topicFoundInSentence = [];
 	let markings = [];
-	const indexOfSentence = 0;
-	let indexRunningThroughSentence = 0;
-	const matchesIndices = [];
 
 	sentences.forEach( function( sentence ) {
 		topicFoundInSentence = matchWords( sentence, wordsToMark, locale ).matches;
 
 		if ( topicFoundInSentence.length > 0 ) {
-			topicFoundInSentence.forEach( function( occurrence ) {
-				const indexOfOccurrenceInSentence = sentence.indexOf( occurrence, indexRunningThroughSentence );
```

Comment:
According to a commit comment, this code is removed because it is not used anymore.
I think you forgot it to add to the PR as a relevant technical choice 😉.